Example/Pods/Local Podspecs/VOKUtilities.podspec.json

```diff
@@ -29,40 +34,65 @@
       "source_files": "Pod/NSString+VOKValidation/*.{h,m}"
     },
     {
-      "name": "VOKIBHelpers",
-      "source_files": "Pod/VOKIBHelpers/*.{h,m}"
-    },
-    {
       "name": "NSPredicate+VOKAL",
       "source_files": "Pod/NSPredicate+VOKAL/*.{h,m}"
     },
     {
+      "name": "NSNumberFormatter+VOKAL",
+      "source_files": "Pod/NSNumberFormatter+VOKAL/*.{h,m}"
+    },
+    {
+      "name": "UIColor+VOKAL",
+      "platforms": {
+        "ios": "7.0",
+        "tvos": "9.0"
+      },
+      "source_files": "Pod/UIColor+VOKAL/*.{h,m}"
+    },
+    {
+      "name": "VOKIBHelpers",
+      "platforms": {
+        "ios": "7.0",
+        "tvos": "9.0"
+      },
+      "source_files": "Pod/VOKIBHelpers/*.{h,m}"
+    },
+    {
       "name": "UIView+VOKDebug",
+      "platforms": {
+        "ios": "7.0",
+        "tvos": "9.0"
+      },
       "source_files": "Pod/UIView+VOKDebug/*.{h,m}"
     },
     {
       "name": "UIView+VOKCircle",
+      "platforms": {
+        "ios": "7.0",
+        "tvos": "9.0"
+      },
       "source_files": "Pod/UIView+VOKCircle/*.{h,m}"
     },
     {
       "name": "UIViewController+VOKKeyboard",
+      "platforms": {
+        "ios": null
```

Comment:
I missed that this is the local podspec from CocoaPods, so it really doesn't matter